service.json

```diff
@@ -1,6 +1,6 @@
 {
   "downloads1": {
-    "module": "downloads",
+    "module": "@npm/downloads",
```

Comment:
I would love it if `@npm` was on registry2 so we could view our public packages on the site. Is this possible/easy?